commands/pack.js

```diff
@@ -57,6 +57,7 @@ function displayHelp() {
 	console.log();
 	/*
 		Private Options:
+			--additional-entry      Split JS/CSS bundles into additional entry files
```

Comment:
Do you see "--entry" option right below your added code?
I think it's not different one. Why don't we use the exsiting one?

Reply:
I checked, `--entry` takes only one js file to be set as entry and replaces it with the parameter `replacement`.
https://github.com/enactjs/cli/blob/6bb7e219ba112331e53dff54c9fe01addbcb6a21/commands/pack.js#L266
It cannot be used for code split that we have set as DoD.

Should I change the code to satisfy DoD with this `--entry` option?

Reply:
No, definately not.
I was saying that 'entry' seems similar build option to split code so if it's possible to extend the functionality, it would be better using this instead of adding another build option.